apps/src/templates/CertificateShare.jsx

```diff
@@ -1,20 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import {TwoColumnActionBlock} from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
+import styleConstants from '../styleConstants';
 
 export default function CertificateShare(props) {
+  const {announcement} = props;
   return (
-    <a href={props.printUrl}>
-      <img
-        src={props.imageUrl}
-        alt={i18n.certificateForCompletion()}
-        width="100%"
+    <div style={styles.wrapper}>
+      <a href={props.printUrl}>
+        <img
+          src={props.imageUrl}
+          alt={i18n.certificateForCompletion()}
+          width="100%"
+          style={styles.certificate}
+        />
+      </a>
+      <TwoColumnActionBlock
+        isRtl={false}
+        imageUrl={pegasus(announcement.image)}
+        subHeading={announcement.title}
+        description={announcement.body}
+        buttons={[
+          {
+            id: announcement.buttonId,
+            url: announcement.buttonUrl,
+            text: announcement.buttonText
+          }
+        ]}
       />
-    </a>
+    </div>
   );
 }
 
+const styles = {
+  wrapper: {
+    with: '100%',
+    maxWidth: styleConstants['content-width'],
+    marginLeft: 'auto',
+    marginRight: 'auto'
+  },
+  certificate: {
+    marginBottom: 20
+  }
+};
+
 CertificateShare.propTypes = {
   imageUrl: PropTypes.string.isRequired,
-  printUrl: PropTypes.string.isRequired
+  printUrl: PropTypes.string.isRequired,
+  announcement: PropTypes.object.isRequired
```

Comment:
Is this required? Will there always be an announcement?

Reply:
Great call Maureen! I confirmed that this is enough to let the page still render properly when the corresponding entry is removed from announcements.json.
![Screen Shot 2022-02-15 at 12 02 48 PM](https://user-images.githubusercontent.com/8001765/154140140-672af7fb-7dd5-46ae-9a9e-c9f965035687.png)